Gordon360/Models/ViewModels/RecIM/TeamPatchViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Drawing.Text;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class TeamPatchViewModel
+    {
+        public int ID { get; set; }
```

Comment:
```suggestion
```
Remove ID, we will be passing in ID as a parameter in the route and PATCH will use that parameter to query